tests/cmd/fs-idmap/fs-idmap.go

```diff
@@ -1,3 +1,5 @@
+// Package main provides a command-line tool which detects if a filesystem
```

Comment:
It looks like you don't have to write such nonsense as "Package main ...".

I tried this and did not see any issue with `golangci-lint run --config .golangci-extra.yml`:

```
// Command fs-idmap is a tool to detect if a filesystem associated with                                                   
// a given path supports id-mapped mounts. Used by runc integration tests. 
```

Maybe such exception exists for "main" packages only, but it totally makes sense.

Reply:
Looking into Go tools repo, I see that they don't use "Package main ..." either. For example, this is goimports' doc.go, the package-level docstring starts with "Command goimports ...": https://github.com/golang/tools/blob/master/cmd/goimports/doc.go

Reply:
Here's how it is rendered: https://pkg.go.dev/golang.org/x/tools/cmd/goimports

Reply:
Okay, I'll rewrite these ones.